packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -40,24 +41,65 @@ const releaseController = {
   async findOne(ctx: Koa.Context) {
     const id: GetRelease.Request['params']['id'] = ctx.params.id;
 
-    const result = (await getService('release', { strapi }).findOne(
-      Number(id)
+    const allowedContentTypes = getAllowedContentTypes({ strapi, userAbility: ctx.state.userAbility });
+
+    const contentManagerContentTypeService = strapi
+      .plugin('content-manager')
+      .service('content-types');
+
+    const contentTypesMeta = await allowedContentTypes.reduce(async (accumulatorPromise, contentTypeUid) => {
+      const acc = await accumulatorPromise;
+
+      const contentTypeConfig = await contentManagerContentTypeService.findConfiguration({ uid: contentTypeUid });
+
+      if (contentTypeConfig) {
+        acc[contentTypeUid] = {
+          mainField: contentTypeConfig.settings.mainField,
+        };
+      }
+
+      return acc;
+    }, Promise.resolve({} as Record<UID.ContentType, { mainField: string }>));
+
+    const releaseWithCountAllActions = (await getService('release', { strapi }).findOne(
+      Number(id), { populate: { actions: { count: true } } }
+    )) as ReleaseWithPopulatedActions | null;
+
+    const releaseWithCountHiddenActions = (await getService('release', { strapi }).findOne(
```

Comment:
what are hidden actions?

Reply:
Ah I see now, the content types the user doesnt have visibility on.
Extending from my previous comment, could we do something like:
```ts
getService('action', { strapi }).count(releaseId, {filters: { contentType: { $notIn: allowedContentTypes }} )
````